projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/ui-handler/with-ui-handler-data-provider.tsx

```diff
@@ -101,8 +102,8 @@ const withUiHandlerDataProvider = createHigherOrderComponent( BlockListBlock =>
 		 *
 		 * @returns {void}
 		 */
-		const selectFormBlock = useCallback( () => {
-			dispatch( 'core/block-editor' ).selectBlock( props.clientId );
+		const selectFormBlock = useCallback( async () => {
+			return dispatch( 'core/block-editor' ).selectBlock( props.clientId );
```

Comment:
`async` is only necessary if there is any `await`, so it should not be necessary here as a promise is already returned.

Reply:
needed for this: https://github.com/Automattic/jetpack/pull/32262/files#diff-4a89b9060d3c34a2791611bb74f0ba9bc5c295ffda62f7ddd2b2704f6b5443fcR268

Reply:
you're right.